### DISCUSSION TITLE
Cloning from github.com not working on IPv6-only host

### DISCUSSION THREAD

**Discussion by rfolo9li** [community/community#45744] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,
When I execute `git clone` on a host which uses IPv6 only, then I get the error "Couldn't connect to server".

Example:
`git clone https://github.com/silverstripe/silverstripe-installer.git/`

> Cloning into 'silverstripe-installer'...
> fatal: unable to access 'https://github.com/silverstripe/silverstripe-installer.git/': Couldn't connect to server

In my opinion, it is a serious bug because it renders GitHub unusable on hosts which have IPv6 only. There is already a [discussion](https://github.com/community/community/discussions/10539) about this topic but no reaction from GitHub. So I am filing this as a bug hoping the issue will get the needed attention as today, many internet service providers just provide IPv6 for there are no IPv4 addresses left (or demand extra fees for IPv4 addresses).

**Comment by jge162** [comment]
Hello @rfolo9li thank you for your post, this has been well documented for awhile now, not eta given yet. You can see old post from last year here -> [post](https://github.com/orgs/community/discussions/10539)
